src/stats.rs

```diff
@@ -51,12 +224,6 @@ pub struct Event {
 
     /// The value being reported. Meaning differs based on event name.
     value: i64,
-
-    /// The client or server connection reporting the event.
-    process_id: i32,
-
-    /// The server the client is connected to.
-    address_id: usize,
```

Comment:
These are no longer needed, we are using anonymous structs to attach information specific to each event.